doc/api/readline.md

```diff
@@ -349,6 +349,55 @@ async function processLineByLine() {
 }
 ```
 
+### rl.line
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
```

Comment:
```suggestion
* {string|undefined}
```

If I understand it correctly, this is not defined in the non-TTY case?

Reply:
You're absolutely right.  Looks like it's the same for `rl.cursor`, too.  Thanks for catching that.